src/crypto/tls/common.go

```diff
@@ -673,8 +709,23 @@ type Config struct {
 	// used for debugging.
 	KeyLogWriter io.Writer
 
+	// SupportDelegatedCredential is true if the client or server is willing
+	// to negotiate the delegated credential extension.
+	// This can only be used with TLS 1.3.
+	//
+	// See https://tools.ietf.org/html/draft-ietf-tls-subcerts.
+	SupportDelegatedCredential bool
```

Comment:
1. This field indicates whether to support DCs, right?  In that case the missing "s" is correct.
2. Doesn't GetDelegatedCredential already flag this in the case we're sending?  So perhaps the field should rather be "AcceptDelegatedCredential".  Why make this configurable?

Reply:
No. How it works is:

1. If ServerAuth: The client advertises that they support delegated credentials by using the `SupportDelegatedCredential` field. The server gets its delegated credential by using `GetDelegatedCredential` and sends it along in the certificate.
1. If ClientAuth: The server advertises that they support delegated credentials by using the `SupportDelegatedCredential` field. The client gets its delegated credential by using `GetDelegatedCredential` and sends it along in the certificate.

Reply:
what 's'?

Reply:
I see what you mean with the 's' now.

Reply:
I'm still confused: why shouldn't a client that supports delegated credentials (like the one we're writing now) unconditionally use it.  Why make disabling it configurable?

Reply:
Let's talk in the call, cause I'm not following ;)

Reply:
I'll add the todo as discussed in the call.

Reply:
In this case, I don't think the 's' should be added. The other fields does not have an 's', as seen in `GetCertificate`, `GetClientCertificate`.